packages/stepper/src/InternalStep.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { PropsWithChildren } from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { StepStates, InternalStepProps } from './types';
+import StepIcon from './StepIcon';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+import { Body } from '@leafygreen-ui/typography';
+import { useStepperContext } from './StepperContext';
+
+export const Step = function Step({
+  children,
+  index,
+  state,
+  ariaLabel = `step${index || ''}`,
+  shouldDisplayLine = true,
+  iconSize = 20,
+  className,
+  ...rest
+}: PropsWithChildren<InternalStepProps & React.HTMLProps<HTMLDivElement>>) {
+  const {
+    isDarkMode,
+    stepIconClassName,
+    stepLabelClassName,
+  } = useStepperContext();
+  const isCurrent = state === StepStates.Current;
+
+  const baseStyles = css`
```

Comment:
Same comment as before, we shouldn't define styles within render unless absolutely necessary

Reply:
I think you meant inside the component, not within render? I do need the `isDarkMode` variable though, so I'll use it here.